applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -718,6 +711,19 @@ int unicast_server_enable(le_audio_receive_cb recv_cb)
 
 	channel_assignment_get(&channel);
 
+	if (!IS_ENABLED(CONFIG_BT_CSIP_SET_MEMBER_TEST_SAMPLE_DATA)) {
+		if (!strcmp(CONFIG_BT_SET_IDENTITY_RESOLVING_KEY_DEFAULT,
+			    CONFIG_BT_SET_IDENTITY_RESOLVING_KEY)) {
+			LOG_WRN("CSIP using the default SIRK, must be changed before production");
+		}
+
+		memcpy(csip_param.set_sirk, CONFIG_BT_SET_IDENTITY_RESOLVING_KEY,
+		       MIN(sizeof(CONFIG_BT_SET_IDENTITY_RESOLVING_KEY),
+			   sizeof(csip_param.set_sirk)));
+	} else {
+		LOG_WRN("CSIP test sample data is used, must be changed before production");
+	}
+
```

Comment:
For readability:
```suggestion
	if (IS_ENABLED(CONFIG_BT_CSIP_SET_MEMBER_TEST_SAMPLE_DATA)) {
		LOG_WRN("CSIP test sample data is used, must be changed before production");		
	} else {
	        if (!strcmp(CONFIG_BT_SET_IDENTITY_RESOLVING_KEY_DEFAULT,
			    CONFIG_BT_SET_IDENTITY_RESOLVING_KEY)) {
			LOG_WRN("CSIP using the default SIRK, must be changed before production");
		}

		memcpy(csip_param.set_sirk, CONFIG_BT_SET_IDENTITY_RESOLVING_KEY,
		       MIN(sizeof(CONFIG_BT_SET_IDENTITY_RESOLVING_KEY),
			   sizeof(csip_param.set_sirk)));

	}

```

Reply:
I had it like that but it looked a bit messy. I will go with == 0.